examples/k8s_statefulsets/README.md

```diff
@@ -23,7 +23,13 @@ $ minikube start --cpus=2 --memory=2040 --vm-driver=virtualbox
 $ kubectl create namespace test-rabbitmq
 ```
 
-* Deploy the  `YAML` file:
+* Deploy RBAC `YAML` file:
+
+```
+$ kubectl create -f examples/k8s_statefulsets/rabbitmq_rbac.yaml
+```
```

Comment:
Can you please add a note here? 
why we need `rbac`,  something like you already wrote here:
```
For kubernetes 1.6 or above, it enabled RBAC by default
```

Reply:
added as you suggested.